_includes/guide.njk

```diff
@@ -50,35 +50,43 @@ layout: layout
   </div>
 </main>
 
-{# <div class="cta-next-guide">
+{% set next = false %}
+{% for item in collections[tags[0]] %}
```

Comment:
Would this work if we had more than one tag? Or is this a temporary solution to test the idea?

Reply:
I'm not sure if this will work because there's no way to ensure an order to the items in the collection. I think they are ordered by their author date.

Reply:
This was the reason why I asked if we allow more than one collection. I'm not sure we'll need this code at all, if we scrape the "next guide" link, but wanted to show it here to discuss exactly this. 

As for ordering, there is a way to order the collection other than by date, but its tricky. So first we should answer the question - do we allow post to belong to multiple freeform collections and then we don't have linear navigation in any of them, or we allow 1 "path" post (and then it can be ordered arbitrarily)?

Reply:
I think we'll want to only allow a post to live in one collection.

Sorry I meant, if we have a collection of posts tagged 'accessible' and a different collection of posts tagged 'speed', won't `tags[0]` always equal 'accessible' and thus never show posts from 'speed'?

Reply:
I think my concern about ordering is it would probably require a post having an `order` field in its YAML, and at that point I feel like the benefits of using a tag for the collection start to go down because you can't quickly get a look at the table of contents for the whole collection.

I think this is why I ended up using a directory data file instead to organize the table of contents for a particular collection because it consolidated everything into one place.